assets/text/message_data.h

```diff
@@ -0,0 +1 @@
+#include "text/message_data.h"
```

Comment:
In view of adding includes below this one for modding, the vanilla text debugger won't recognize them anymore since it stops search at message `FFFD`. I suggest adding messages 0xFFFC and 0xFFFD to this header instead of in the extracted header

Reply:
If we still want to extract them, could possibly just add them to a separate `message_data_nes.h` that gets included below `message_data.h`. Then custom text data could go in between.

Reply:
Yeah I was considering a footer include as well as an option, extracted or not

Not sure yet what to do, anyone feel free to chip in
I won't get to this immediately

Reply:
I think including them in the committed header works fine

Reply:
d75522b243d1f43171fc5f2b574cc976b85c7f43 Move messages 0xFFFC, 0xFFFD to committed message_data.h

and explanation comment